PhysicsTools/NanoAOD/python/photons_cff.py

```diff
@@ -174,12 +173,17 @@ def make_bitmapVID_docstring(id_modules_working_points_pset):
     doc = cms.string("slimmedPhotons after basic selection (" + finalPhotons.cut.value()+")"),
     singleton = cms.bool(False), # the number of entries is variable
     extension = cms.bool(False), # this is the main table for the photons
-    variables = cms.PSet(CandVars,
+    variables = cms.PSet(P3Vars,
         jetIdx = Var("?hasUserCand('jet')?userCand('jet').key():-1", int, doc="index of the associated jet (-1 if none)"),
         electronIdx = Var("?hasUserCand('electron')?userCand('electron').key():-1", int, doc="index of the associated electron (-1 if none)"),
         energyErr = Var("getCorrectedEnergyError('regression2')",float,doc="energy error of the cluster from regression",precision=6),
+        energyRaw = Var("superCluster().rawEnergy()",float,doc="raw energy of photon supercluster", precision=10),
         r9 = Var("full5x5_r9()",float,doc="R9 of the supercluster, calculated with full 5x5 region",precision=10),
         sieie = Var("full5x5_sigmaIetaIeta()",float,doc="sigma_IetaIeta of the supercluster, calculated with full 5x5 region",precision=10),
+        sieip = Var("full5x5_showerShapeVariables().sigmaIetaIphi",float,doc="sigma_IetaIphi of the supercluster, calculated with full 5x5 region",precision=10),
```

Comment:
add also the sipip so that useful to remove "spikes" see here
https://indico.cern.ch/event/1085967/contributions/4565988/attachments/2335155/3980120/Report_2l2nu_NanoAOD_20211027.pdf
also do we need these 3 shower shape with full float precision ?